### PR TITLE
[Office Add-ins] Create blog post for June community call

### PR DESCRIPTION
**Pull request by samantharamon**
## Category

- [ ] Content fix
- [X] New article

## Related issues

N/A

## Contents of the Pull Request

Blog post for the Office Add-ins June community call.

## Guidance

None

### CODE REVIEW

File: content/office-add-ins-community-call/2022-06-08/index.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+* Update – Microsoft roadmap for One Outlook – [Nikita Mittal](https://www.linkedin.com/in/nikita-mittal-92536648) – Senior Product Manager (Microsoft) – [1:23](https://www.youtube.com/watch?v=aIydcCxJFBo&t=83s)
+* Demo – Microsoft Outlook Smart Alerts – [Sam Ramon](https://github.com/samantharamon) – Technical Writer (Microsoft) – [7:35](https://www.youtube.com/watch?v=aIydcCxJFBo&t=455s) 
+* Demo – Using Blazor in a Microsoft Excel add-in – [Maarten van Stam](http://twitter.com/aafvstam) – MVP Office Add-ins \| @aafvstam – [15:12](https://www.youtube.com/watch?v=aIydcCxJFBo&t=912s)
+* Community spotlight – [32:26](https://www.youtube.com/watch?v=aIydcCxJFBo&t=1946s)
+* Community panels – [33:10](https://www.youtube.com/watch?v=aIydcCxJFBo&t=1990s)
+* Q&A – [David Chesnut](http://twitter.com/davidchesnut) – Senior Technical Writer (Microsoft) \| @davidchesnut – [36:26](https://www.youtube.com/watch?v=aIydcCxJFBo&t=2186s)
+
+## Actions
+
+* Give us feedback on what you need from code samples. Join the Teams call – [Office Add-ins samples discussion](https://aka.ms/officeaddinsamples-call) – June 15, 8:00 AM PT \| <https://aka.ms/officeaddinsamples-call>
```
Comment (davidchesnut) [inline content/office-add-ins-community-call/2022-06-08/index.md:49]:
```suggestion
* Give us feedback on what you need from code samples. Join the Teams call – [Office Add-ins samples discussion](https://aka.ms/officeaddinsamples-join) – June 15, 8:00 AM PT \| <https://aka.ms/officeaddinsamples-call>
```

Updating with link to the calendar invite.

**Review by davidchesnut — APPROVED** [review]
One small URL change, otherwise looks great!

**Comment by lindalu-MSFT** [comment]
@samantharamon Is there a preview link I can look at?

File: content/office-add-ins-community-call/2022-06-08/index.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+    * Blog - [Soft As In Software blog](https://blog.softasinsoftware.com/)
+    * Website - [Maarten van Stam – Soft As In Software](https://www.softasinsoftware.com/)
+
+## General resources
+
+* [Office developer center](https://developer.microsoft.com/office)
+
+## Stay connected
+
+* See the full blog post for this call in the Microsoft 365 platform community blog – <https://aka.ms/m365pnp/blog>
```
Comment (lindalu-MSFT) [inline content/office-add-ins-community-call/2022-06-08/index.md:142]:
```suggestion
* See the full blog post for this call in the [Microsoft 365 platform community blog](https://aka.ms/m365pnp/blog)
```
For SEO, suggest you make these actual links for search engines to crawl the link text for keywords.

File: content/office-add-ins-community-call/2022-06-08/index.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+* Twitter – <https://twitter.com/microsoft365dev>
```
Comment (lindalu-MSFT) [inline content/office-add-ins-community-call/2022-06-08/index.md:143]:
```suggestion
* Follow us on [Twitter](https://twitter.com/microsoft365dev)
```

File: content/office-add-ins-community-call/2022-06-08/index.md [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+* Microsoft 365 Platform Community in YouTube – <https://aka.ms/m365/videos>
```
Comment (lindalu-MSFT) [inline content/office-add-ins-community-call/2022-06-08/index.md:144]:
```suggestion
* [Microsoft 365 Platform Community on YouTube](https://aka.ms/m365/videos)
```

File: content/office-add-ins-community-call/2022-06-08/index.md [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+* Link to all Microsoft Developer Community calls – <https://aka.ms/M365DevCalls>
```
Comment (lindalu-MSFT) [inline content/office-add-ins-community-call/2022-06-08/index.md:145]:
```suggestion
* All [Microsoft Developer Community calls](https://aka.ms/M365DevCalls)
```

File: content/office-add-ins-community-call/2022-06-08/index.md [hunk 6]
```diff
@@ -0,0 +1,149 @@
...
+* Submit questions for next community call – <https://aka.ms/officeaddinsform>
```
Comment (lindalu-MSFT) [inline content/office-add-ins-community-call/2022-06-08/index.md:146]:
```suggestion
* [Submit questions about add-ins development](https://aka.ms/officeaddinsform) for our next community call
```

File: content/office-add-ins-community-call/2022-06-08/index.md [hunk 7]
```diff
@@ -0,0 +1,149 @@
...
+* Next community call – July 13th at 8:00 AM PT – <https://aka.ms/officeaddinscommunitycall>
```
Comment (lindalu-MSFT) [inline content/office-add-ins-community-call/2022-06-08/index.md:147]:
```suggestion
* Next community call – July 13th at 8:00 AM PT – [Add to calendar](https://aka.ms/officeaddinscommunitycall)
```

File: content/office-add-ins-community-call/2022-06-08/index.md [hunk 8]
```diff
@@ -0,0 +1,149 @@
...
+## Actions
+
+* Give us feedback on what you need from code samples. Join the Teams call – [Office Add-ins samples discussion](https://aka.ms/officeaddinsamples-join) – June 15, 8:00 AM PT \| <https://aka.ms/officeaddinsamples-call>
+* Share your feedback on how we can provide you with a better Office Add-ins development experience. Join a community panel –
+    * [Outlook add-ins panel](https://ux.microsoft.com/Panel/OutlookAddinDeveloper)
+    * [Excel add-ins panel](https://ux.microsoft.com/Panel/ExcelAddinDeveloper)
+    * [Word add-ins panel](https://ux.microsoft.com/Panel/WordAddinDeveloper)
+    * [PowerPoint add-ins panel](https://ux.microsoft.com/Panel/PowerPointAddinDeveloper)
+    * [Samples and docs](https://ux.microsoft.com/Panel/OfficeAddinImproveSamplesDocs)
+* Sign up for the PnP Recognition Program and earn contribution badges – <https://pnp.github.io/recognitionprogram/>
```
Comment (lindalu-MSFT) [inline content/office-add-ins-community-call/2022-06-08/index.md:56]:
```suggestion
* [Sign up for the PnP Recognition Program](https://pnp.github.io/recognitionprogram/) and earn contribution badges
```

File: content/office-add-ins-community-call/2022-06-08/index.md [hunk 9]
```diff
@@ -0,0 +1,149 @@
...
+This month's community call topics:
+
+* **Microsoft roadmap for One Outlook** – Get to know more about One Outlook, including enabling support for web add-ins and the planned timeline for preview and release – [Nikita Mittal](https://www.linkedin.com/in/nikita-mittal-92536648) – Senior Product Manager (Microsoft)
+* **Microsoft Outlook Smart Alerts** – Preview how to use Outlook Smart Alerts to enable your add-in to run logic when a user sends a message or appointment – [Sam Ramon](https://github.com/samantharamon) – Technical Writer (Microsoft)
+* **Using Blazor in a Microsoft Excel add-in** – Step through creating a Blazor Webassembly Excel add-in – [Maarten van Stam](http://twitter.com/aafvstam) – MVP Office Add-ins \| @aafvstam
+* **Community spotlight** recognizes [Maarten van Stam](http://twitter.com/aafvstam).
+* **Community panels** – Join us to share your feedback on how we can provide you with a better Office Add-ins development experience.
+* **Q&A** – See question and answers at end of call and in chat throughout call.
+
+The call was hosted by [David Chesnut](http://twitter.com/davidchesnut) (Microsoft) \| @davidchesnut. Recorded June 8, 2022.
```
Comment (lindalu-MSFT) [inline content/office-add-ins-community-call/2022-06-08/index.md:28]:
```suggestion
The call was hosted by [David Chesnut](http://twitter.com/davidchesnut) (Microsoft) | @davidchesnut. Recorded June 8, 2022.
```

File: content/office-add-ins-community-call/2022-06-08/index.md [hunk 10]
```diff
@@ -0,0 +1,149 @@
+---
+title: "Office Add-ins community call – June 8, 2022"
+summary: "Topics: Microsoft roadmap for One Outlook, Microsoft Outlook Smart Alerts (Demo), Using Blazor in a Microsoft Excel add-in. Q&A in chat and at end of call."
+date: 2022-06-13T08:01:00-05:00
+author: "Sam Ramon"
+githubname: samantharamon
+categories: ["Office add-in developer community call"]
```
Comment (lindalu-MSFT) [inline content/office-add-ins-community-call/2022-06-08/index.md:7]:
```suggestion
categories: ["Office Add-in developer community call"]
```

**Review by lindalu-MSFT — CHANGES_REQUESTED** [review]

File: content/office-add-ins-community-call/2022-06-08/index.md [hunk 11]
```diff
@@ -0,0 +1,149 @@
...
+draft: true
+---
+
+## Call Summary
+
+This month's community call topics:
+
+* **Microsoft roadmap for One Outlook** – Get to know more about One Outlook, including enabling support for web add-ins and the planned timeline for preview and release – [Nikita Mittal](https://www.linkedin.com/in/nikita-mittal-92536648) – Senior Product Manager (Microsoft)
+* **Microsoft Outlook Smart Alerts** – Preview how to use Outlook Smart Alerts to enable your add-in to run logic when a user sends a message or appointment – [Sam Ramon](https://github.com/samantharamon) – Technical Writer (Microsoft)
+* **Using Blazor in a Microsoft Excel add-in** – Step through creating a Blazor Webassembly Excel add-in – [Maarten van Stam](http://twitter.com/aafvstam) – MVP Office Add-ins \| @aafvstam
```
[the remark this answers is not in the document]
Comment (samantharamon) [reply content/office-add-ins-community-call/2022-06-08/index.md:23]: I initially thought this was used to escape the pipe character in Markdown, but it looks like we don't need to do this for it. Committing your suggestion.

**Comment by samantharamon** [comment]
> @samantharamon Is there a preview link I can look at?

Unfortunately, there doesn't appear to be a way to preview the content as it would appear on the blog. For now, this is the best way to preview the content in GitHub: https://github.com/pnp/blog/blob/084760a0bdd9591a3339ed8116abedb5d50a2582/content/office-add-ins-community-call/2022-06-08/index.md

**Review by lindalu-MSFT — APPROVED** [review]

**Comment by lindalu-MSFT** [comment]
@samantharamon Approved!

**Pull request merged**